### PR TITLE
Fixes space wind affecting freon ice slipping

### PR DESCRIPTION
**Pull request by Iamgoofball**
this is why we dont question BYOND

### CODE REVIEW

File: code/game/turfs/open.dm [hunk 1]
```diff
@@ -145,7 +145,7 @@
 			C.slipping = TRUE
```
Comment (xxalpha) [inline code/game/turfs/open.dm]: Setting slipping to TRUE here does nothing, because it is immediately set to FALSE after.

Comment (Iamgoofball) [inline code/game/turfs/open.dm]: i still can't reproduce space wind pushing during slips with either of the configurations

can someone try and reproduce it

Comment (MrPerson) [inline code/game/turfs/open.dm]: Setting slipping to false afterwards is the same as just changing this line to C.slipping = FALSE and removing the line below outright

**Comment by optimumtact** [comment]
This literally does nothing

**Pull request closed without merging**